packages/pwa-kit-extension-sdk/src/configs/babel/utils.ts

```diff
@@ -36,9 +35,8 @@ const PLACEHOLDER_PATH = `${NODE_MODULES_PATH}/@salesforce/pwa-kit-extension-sdk
  * developers' environments working on templates with extensibility. Avoids relative paths
  * or symlinked paths that Babel doesn't support by using realpathSync.
  */
-export const buildBabelExtensibilityArgs = () => {
-    const extensions = getConfiguredExtensions(getConfig())
-
+export const buildBabelExtensibilityArgs = (config: any) => {
+    const extensions = getConfiguredExtensions(config)
```

Comment:
One of the main changes in the latest commits is making sure that the extension-sdk do not depend on the runtime package. So in this case, I replaced the `getConfig` call with a config parameter.